container/Exchange.js

```diff
@@ -49,11 +53,19 @@ export default () => {
 
   return (
     <Container>
-      {loading || !translation ? (
-        <Spinner />
-      ) : (
-        <Translation translationResult={translation} />
-      )}
+      {(() => {
+        let element = null
+        if (search === undefined) {
```

Comment:
This is the same as (!loading && !translation). We could bring back the if-else 4-way path that we had before

Reply:
Actually when you load the website with no search queries, `loading` is defaulted to `true`, hence why I had to check `search` instead; `!loading && !translation` would never equal true in the default search case of no search params

Reply:
Maybe loading should default to false. That would make more sense, and only set loading to true if we actually do a search

Reply:
Did so in the latest commit but I still put in a `|| search === ''`, so really is there any gain in doing `(!loading && !translation) || search === ''` over `!search`?

Reply:
I agree

Reply:
In a later PR we could refactor this into a state enum like this https://kentcdodds.com/blog/stop-using-isloading-booleans

We could also add error handling